### ISSUE TITLE
VisualEffects manifest parsing error

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#503]
```
2020-07-02T14:06:37.915544941Z [ INFO ] pulling https://github.com/twostraws/VisualEffects.git in /checkouts/github.com-twostraws-visualeffects
2020-07-02T14:06:43.157994028Z [ INFO ] listing tags for package https://github.com/twostraws/VisualEffects.git
2020-07-02T14:06:46.139437582Z path:    "/checkouts/github.com-twostraws-visualeffects"
2020-07-02T14:06:46.139459588Z Message: "/checkouts/github.com-twostraws-visualeffects: error: manifest parse error(s):
2020-07-02T14:06:46.139549182Z path:    "/checkouts/github.com-twostraws-visualeffects"
2020-07-02T14:06:46.139578424Z Message: "/checkouts/github.com-twostraws-visualeffects: error: manifest parse error(s):
2020-07-02T14:06:46.140797870Z path:    "/checkouts/github.com-twostraws-visualeffects"
2020-07-02T14:06:46.141012748Z Message: "/checkouts/github.com-twostraws-visualeffects: error: manifest parse error(s):
2020-07-02T14:06:46.142574051Z path:    "/checkouts/github.com-twostraws-visualeffects"
2020-07-02T14:06:46.142941354Z Message: "/checkouts/github.com-twostraws-visualeffects: error: manifest parse error(s):
2020-07-02T14:06:46.156207449Z [ ERROR ] analysis error: No valid version found for package 'https://github.com/twostraws/VisualEffects.git' (id: Optional(DF37C56B-D0E5-4452-9CDC-A628DE3D42AE))
```

**Comment by finestructure** [comment]
```
root@7b3c2b698267:/checkouts/github.com-twostraws-visualeffects# swift package dump-package
/checkouts/github.com-twostraws-visualeffects: error: manifest parse error(s):
<unknown>:0: error: module compiled with Swift 5.2.4 cannot be imported by the Swift 5.3 compiler: /usr/lib/swift/linux/x86_64/Swift.swiftmodule
```

**Comment by finestructure** [comment]
This is because we copy over runtime bits to support running the analysis binary - and that kills our ability to run `swift package dump-package`. We need to roll this back, because we’re now unable to analyse anything :/

**Comment by daveverwer** [comment]
Closed by [PackageList#465](https://github.com/SwiftPackageIndex/PackageList/pull/465).
